### PR TITLE
[TransactionsPage] functional components, hooks & css modules

### LINKED ISSUE #2630: TransactionsPage: functional components, css modules & hooks refactor
This is a sub-issue from #2438 & #2439, meant to refactor components from the `views/TransactionsPage` section towards a functional approach. Also refactor css to modules and insert hooks instead of connectors.

### PR DESCRIPTION
**Pull request by thi4go**
This is another diff towards #2438 and #2439. It implements the ongoing refactor for the `TransactionsPage` view and its components.

closes #2630

before dir:
Screenshot from 2020-08-31 19-10-50

after dir:
newdir

### CODE REVIEW

**Comment by vctt94** [comment]
needs rebase

File: app/components/views/TransactionsPage/SendTab/SendTab.jsx [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+      }
+    }];
+    setIsSendAll(false);
+    onSetOutputs(newOutputs);
+    onAttemptConstructTransaction();
+  };
+
+  const hasError = useCallback(() => {
+    let hasError = false;
+    outputs.forEach((o) => {
```
Comment (victorgcramos) [inline app/components/views/TransactionsPage/SendTab/SendTab.jsx:179]: I'd use the piece of code below to improve readability
```javascript
const outputHasError = (o) =>!o.data.amount ||
        o.data.destination.length === 0 ||
        o.data.error.amount ||
        o.data.error.address
return outputs.some(outputHasError);
```

**Review by victorgcramos — COMMENTED** [review]
Code looks good to me! Left just one inline comment!

File: app/components/views/TransactionsPage/SendTab/SendTab.jsx (hunk 1 above)
Comment (thi4go) [reply app/components/views/TransactionsPage/SendTab/SendTab.jsx:179]: liked the suggestion mate, committed

**Comment by thi4go** [comment]
rebased mate @vctt94

**Review by vctt94 — APPROVED** [review]
Great. LGTM!

**Review by victorgcramos — APPROVED** [review]
LGTM! Some `useCallback` hooks are not necessary because using non-primitive types as dependencies makes the hook to always re-render since non-primitive variables comparison will always be false.

**Review by vctt94 — COMMENTED** [review]
lint is failing

**Review by alexlyp — APPROVED** [review]

**Pull request merged**